pyteal/ast/router.py

```diff
@@ -900,6 +901,13 @@ def _build_program(
             self.contract_construct(),
         )
 
+    @contextmanager
+    def __revert_side_effect(self):
```

Comment:
```suggestion
    def _cleaning_context(self):
```

nit: proposing slightly more unified naming

Reply:
sure, that makes sense

Reply:
Did you inted `_revert_side_effect` rather than `_cleaning_context`. Either is fine (even if I suggested the latter)